### PR TITLE
Added a Note on where to find the database connection string on an existing project

### PR DESCRIPTION
**Pull request by eshanrnh**
## Description

Added a Note on where to find the database connection string on an existing project

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [x] Other

## Product & version (if relevant)

All versions

## Deadline (if relevant)

N/A

### CODE REVIEW

File: 10/umbraco-cms/extending/key-vault.md [hunk 1]
```diff
@@ -33,6 +33,10 @@ dotnet add package Azure.Identity
 
 ### Configuration
 
+{% hint style="info" %}
+You can find the database connection string under the `ConnectionStrings` section in the `appsettings.json` file. For more information, see the [Connection strings settings](../reference/configuration/connectionstringssettings.md) article.
```
Comment (sofietoft) [inline 10/umbraco-cms/extending/key-vault.md:37]: I believe we in most places point to specific config sections using the entire "path" (see my suggestion).
Not sure if we want to use a colon or just the dots. It's a little different how it's used throughout the docs 😅 

```suggestion
You can find the database connection string under Umbraco:CMS:ConnectionStrings section in the `appsettings.json` file. For more information, see the [Connection strings settings](../reference/configuration/connectionstringssettings.md) article.
```

**Review by sofietoft — COMMENTED** [review]
Looks good @eshanrnh ! 💪 

I've added a single comment.

File: 10/umbraco-cms/extending/key-vault.md [hunk 2]
```diff
@@ -33,6 +33,10 @@ dotnet add package Azure.Identity
 
 ### Configuration
 
+{% hint style="info" %}
+You can find the database connection string under Umbraco:CMS:ConnectionStrings section in the `appsettings.json` file. For more information, see the [Connection strings settings](../reference/configuration/connectionstringssettings.md) article.
```
Comment (eshanrnh) [inline 10/umbraco-cms/extending/key-vault.md:37]:
```suggestion
You can find the database connection string under the `Umbraco:CMS:ConnectionStrings` section in the `appsettings.json` file. For more information, see the [Connection strings settings](../reference/configuration/connectionstringssettings.md) article.
```

File: 12/umbraco-cms/extending/key-vault.md [hunk 3]
```diff
@@ -31,6 +31,10 @@ dotnet add package Azure.Identity
 
 ### Configuration
 
+{% hint style="info" %}
+You can find the database connection string under the `ConnectionStrings` section in the `appsettings.json` file. For more information, see the [Connection strings settings](../reference/configuration/connectionstringssettings.md) article.
```
Comment (eshanrnh) [inline 12/umbraco-cms/extending/key-vault.md:35]:
```suggestion
You can find the database connection string under the `Umbraco:CMS:ConnectionStrings` section in the `appsettings.json` file. For more information, see the [Connection strings settings](../reference/configuration/connectionstringssettings.md) article.
```

File: 13/umbraco-cms/extending/key-vault.md [hunk 4]
```diff
@@ -31,6 +31,10 @@ dotnet add package Azure.Identity
 
 ### Configuration
 
+{% hint style="info" %}
+You can find the database connection string under the `ConnectionStrings` section in the `appsettings.json` file. For more information, see the [Connection strings settings](../reference/configuration/connectionstringssettings.md) article.
```
Comment (eshanrnh) [inline 13/umbraco-cms/extending/key-vault.md:35]:
```suggestion
You can find the database connection string under the `Umbraco:CMS:ConnectionStrings` section in the `appsettings.json` file. For more information, see the [Connection strings settings](../reference/configuration/connectionstringssettings.md) article.
```

File: 14/umbraco-cms/extending/key-vault.md [hunk 5]
```diff
@@ -31,6 +31,10 @@ dotnet add package Azure.Identity
 
 ### Configuration
 
+{% hint style="info" %}
+You can find the database connection string under the `ConnectionStrings` section in the `appsettings.json` file. For more information, see the [Connection strings settings](../reference/configuration/connectionstringssettings.md) article.
```
Comment (eshanrnh) [inline 14/umbraco-cms/extending/key-vault.md:35]:
```suggestion
You can find the database connection string under the `Umbraco:CMS:ConnectionStrings` section in the `appsettings.json` file. For more information, see the [Connection strings settings](../reference/configuration/connectionstringssettings.md) article.
```

File: 10/umbraco-cms/extending/key-vault.md (hunk 1 above)
Comment (eshanrnh) [reply 10/umbraco-cms/extending/key-vault.md]: It makes sense to list the entire path 🙌 I believe it's a common practice to use colons when referring to sections within a JSON configuration file.

I've applied the suggestion to all the files.

**Pull request merged**